src/components/CreateProfile.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { Formik, Form, Field, } from 'formik';
+import axios from 'axios';
+import * as Yup from 'yup';
+import { Container, Header, Button } from 'semantic-ui-react';
+
+const CreateProfileSchema =  Yup.object().shape({
+    phone: Yup.number().required("Please enter your phone number"),
+    street: Yup.string(),
+    city: Yup.string().string().required("Please enter your city"),
+    avatarUrl: Yup.string()
+});
+
+
+function CreateProfile (props) {
+
+    return (
+        < Container className='container' >
+            <Header as="h1">Create Profile</Header>
+                <Formik
+                    initialValues = {{ phone: '', street: '', city: '', avatarUrl: '' }}
+                    validationSchema = {CreateProfileSchema}
+                    onSubmit = {( values, { resetForm, setStatus }) => {
+                        axios
+                            .post("https://umts-backend.herokuapp.com/api/auth/profile", values)
```

Comment:
You'll want this to be a `PUT` request instead of a `POST` request since it's updating the profile instead of creating one.